common/src/main/java/bisq/common/app/Version.java

```diff
@@ -45,7 +45,7 @@ public class Version {
      */
     public static final List<String> HISTORICAL_RESOURCE_FILE_VERSION_TAGS = Arrays.asList("1.4.0", "1.5.0", "1.5.2",
             "1.5.5", "1.5.7", "1.6.0", "1.6.3", "1.6.5", "1.7.2", "1.7.4", "1.8.0", "1.8.1", "1.8.3", "1.9.0", "1.9.3",
-            "1.9.5", "1.9.6", "1.9.7", "1.9.10", "1.9.11", "1.9.13");
+            "1.9.5", "1.9.6", "1.9.7", "1.9.10", "1.9.11", "1.9.13", "1.9.15");
```

Comment:
`VERSION = "1.9.15"`
`HISTORICAL...TAGS` would have 1.9.14 (the prior version) appended.